server/middleware/classrooms.coffee

```diff
@@ -230,9 +230,7 @@ module.exports =
     classroom.set 'members', []
     database.assignBody(req, classroom)
 
-    # Copy over data from how courses are right now
-    coursesData = yield module.exports.generateCoursesData(classroom.get('aceConfig')?.language, req.user?.isAdmin())
-    classroom.set('courses', coursesData)
+    yield classroom.setUpdatedCourses(req.user?.isAdmin(), false)
```

Comment:
For things where the arguments aren't descriptive at all, I strongly prefer using named arguments. Eg `setUpdatedCourses(req.user?.isAdmin(), false)` tells me nothing about what the second argument means, but `setUpdatedCourses({ isAdmin: true, addNewCoursesOnly: false })` is pretty clear.